scene/SCsub

```diff
@@ -17,6 +17,7 @@ SConscript("animation/SCsub")
 SConscript("audio/SCsub")
 SConscript("resources/SCsub")
 SConscript("debugger/SCsub")
+SConscript("theme/SCsub")
```

Comment:
I would expect this under `gui` more than its own folder, especially as it's only relevant for `gui` (with the exception of the GUI-adjacent `Window` and `Label3D`).

Reply:
Yeah, and `Theme` is a resource, so it could also be under `resource`... 🙃 I have a plan for another class that could go into `scene/theme`, and we already have `audio` that similarly is in its own folder.